src/applications/check-in/utils/navigation/navigation.unit.spec.js

```diff
@@ -238,6 +238,91 @@ describe('Global check in', () => {
+        ];
+        expect(appointments).to.have.lengthOf(1);
+        const form = updateFormPages(
+          patientDemographicsStatus,
+          testPages,
+          URLS,
+          isTravelReimbursementEnabled,
+          appointments,
+          true,
+          { testId: new Date().toISOString() },
+        );
+        expect(form.find(page => page === URLS.TRAVEL_PAY)).to.be.undefined;
+        expect(form.find(page => page === URLS.TRAVEL_VEHICLE)).to.be.undefined;
+        expect(form.find(page => page === URLS.TRAVEL_ADDRESS)).to.be.undefined;
+        expect(form.find(page => page === URLS.TRAVEL_MILEAGE)).to.be.undefined;
+      });
+      it('should not skip travel pages if facility is in local storage but not in the last day', () => {
+        const patientDemographicsStatus = {
+          demographicsNeedsUpdate: true,
+          demographicsConfirmedAt: '2022-01-04T00:00:00.000-05:00',
+          nextOfKinNeedsUpdate: true,
+          nextOfKinConfirmedAt: '2022-01-04T00:00:00.000-05:00',
+          emergencyContactNeedsUpdate: true,
+          emergencyContactConfirmedAt: '2021-12-01T00:00:00.000-05:00',
+        };
+        const isTravelReimbursementEnabled = true;
+        const appointments = [
+          { stationNo: '0001', clinicIen: '0001', facilityId: 'testId' },
+        ];
+        expect(appointments).to.have.lengthOf(1);
+        const form = updateFormPages(
+          patientDemographicsStatus,
+          testPages,
+          URLS,
+          isTravelReimbursementEnabled,
+          appointments,
+          true,
+          { testId: new Date('2023-01-01T03:24:00').toISOString() },
+        );
+        expect(form.find(page => page === URLS.TRAVEL_QUESTION)).to.exist;
+        expect(form.find(page => page === URLS.TRAVEL_VEHICLE)).to.exist;
+        expect(form.find(page => page === URLS.TRAVEL_ADDRESS)).to.exist;
+        expect(form.find(page => page === URLS.TRAVEL_MILEAGE)).to.exist;
+      });
+      it('should not skip travel pages if facility is not in local storage and there are multiple appointments', () => {
+        const patientDemographicsStatus = {
+          demographicsNeedsUpdate: true,
+          demographicsConfirmedAt: '2022-01-04T00:00:00.000-05:00',
+          nextOfKinNeedsUpdate: true,
+          nextOfKinConfirmedAt: '2022-01-04T00:00:00.000-05:00',
+          emergencyContactNeedsUpdate: true,
+          emergencyContactConfirmedAt: '2021-12-01T00:00:00.000-05:00',
+        };
+        const isTravelReimbursementEnabled = true;
+        const appointments = [
+          { stationNo: '0001', clinicIen: '0001', facilityId: 'testId' },
+          { stationNo: '0001', clinicIen: '0001', facilityId: 'testId' },
+        ];
+        expect(appointments).to.have.lengthOf(2);
+        const form = updateFormPages(
+          patientDemographicsStatus,
+          testPages,
+          URLS,
+          isTravelReimbursementEnabled,
+          appointments,
+          true,
+          {},
```

Comment:
Maybe this test or add another where there is a claim but with a different facility.